pegasus/sites.v3/code.org/views/maker_devices.haml

```diff
@@ -1,13 +1,15 @@
 .action-block__wrapper.action-block__wrapper--two-col
-  .action-block.action-block--one-col
-    %h3 Circuit Playground
-    %img.no-margin-bottom{src: "/images/maker/maker-device-circuit-playground.png", alt: ""}
-    %a.link-button{href: curriculum_link_cp, aria:{label: aria_label_cp}}
-      #{button_label}
-    %a.link-button.secondary{href: "/maker/circuitplayground"} Learn about Circuit Playground
   .action-block.action-block--one-col
     %h3 BBC micro:bit
     %img.no-margin-bottom{src: "/images/maker/maker-device-microbit.png", alt: ""}
     %a.link-button{href: curriculum_link_microbit, aria:{label: aria_label_microbit}}
       #{button_label}
-    %a.link-button.secondary{href: "/maker/microbit"} Learn about micro:bit
+    %a.link-button.secondary{href: "/maker/microbit"}
+      =hoc_s("maker_page.get_started.button.learn_about", markdown: :inline, locals:{device: "micro:bit"})
+  .action-block.action-block--one-col
+    %h3 Circuit Playground
```

Comment:
Since `maker_page.get_started.button.learn_about` was moved to be translatable here, should this string (and "Circuit Playground" below) also be moved?

Reply:
I realized as I was building this page that the names `micro:bit` and `Circuit Playground` shouldn't be translated so I left this as-is.

Reply:
Ah gotcha!